Src/PerformanceCollector/Shared/Implementation/QuickPulse/QuickPulseDataSample.cs

```diff
@@ -69,7 +69,11 @@ public QuickPulseDataSample(QuickPulseDataAccumulator accumulator, IDictionary<s
                 p => (QuickPulseDefaults.CounterOriginalStringMapping.ContainsKey(p.Value.Item1.OriginalString) ? QuickPulseDefaults.CounterOriginalStringMapping[p.Value.Item1.OriginalString] : p.Value.Item1.OriginalString), 
                 p => p.Value.Item2);
 
+            this.TopCpuData = topCpuData.ToArray();
```

Comment:
That's fair, please comment that so some future editor doesn't think they can remove it and thereby accidentally break it.